pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -38,30 +39,34 @@
   typing-extensions,
 }:
 
-# We keep this binary build for two reasons:
+# We keep this binary build for three reasons:
 # - the source build doesn't work on Darwin.
 # - the source build is currently brittle and not easy to maintain
+# - the source build doesn't work on NVIDIA Jetson platforms
```

Comment:
issue link?

Reply:
https://github.com/NixOS/nixpkgs/issues/271077, though will probably be other issues that arise once that is resolved too.

Reply:
ok, let's include a link to that in the comment

Reply:
Same, I think it's OK to add the note in a follow-up PR. This discussion is discoverable via git blame as long as github is online too. Besides we should be generally more aggressive about removing stale comments